app/public/config/config.example.js

```diff
@@ -158,7 +158,14 @@ var config =
 	mobileLastN          : 1,
 	// Highest number of lastN the user can select manually in 
 	// userinteface
-	maxLastN             : 5,
+	maxLastN             : 20,
+	// This values (desktop, mobile) are the limit of visible remote video streams
+	// with full quality. If the number of visible remote video streams is higher
+	// than this value the preferred quality will be decreased, except for
+	// the active speaker, peers pinned to spotlights, videos in new window or videos
+	// in full screen mode. This allows for having up to 20 video windows.
```

Comment:
Perhaps adjust the comment about the 20 video windows as it actually refers to maxLastN and not any of the new variables?  E.g.: "This allows for having more (maxLastN) video windows".

Reply:
I think, that a general reorder of the config variable is a task we must do. All the variables regarding the number of videos should be grouped, and also the settings for simulcast, possible resolutions, as when all depend on each other. But this is a more of a strategic decision.